### PR TITLE
migrate to new places API

### PR DESCRIPTION
**Pull request by birarda**
DO NOT MERGE this until @murillodigital gives the go-ahead

### CODE REVIEW

**Comment by birarda** [comment]
This is ready for review but DO NOT MERGE until @murillodigital confirms that he is ready to merge the data-web side.

File: examples/lobby.js [hunk 1]
```diff
@@ -148,25 +148,25 @@ function drawLobby() {
...
     textures: {}
   };
   
   for (var j = 0; j < NUM_PANELS; j++) {
-    var panelIndex = locationIndexToPanelIndex(j);
-    textureProp["textures"]["file" + panelIndex] = HIFI_PUBLIC_BUCKET + "images/locations/" 
-      + locations[j].id + "/hifi-location-" + locations[j].id + "_640x360.jpg";
+    var panelIndex = placeIndexToPanelIndex(j);
+    textureProp["textures"]["file" + panelIndex] = HIFI_PUBLIC_BUCKET + "images/places/" 
+      + place[j].id + "/hifi-place-" + place[j].id + "_640x360.jpg";
```
Comment (Atlante45) [inline examples/lobby.js]: places[j]

File: examples/lobby.js [hunk 2]
```diff
@@ -252,14 +252,14 @@ function actionStartEvent(event) {
       var panelStringIndex = panelName.indexOf("Panel");
       if (panelStringIndex != -1) {
         var panelIndex = parseInt(panelName.slice(5));
-        var locationIndex = panelIndexToLocationIndex(panelIndex);
-        if (locationIndex < locations.length) {
-          var actionLocation = locations[locationIndex];
+        var placeIndex = panelIndexToPlaceIndex(panelIndex);
+        if (placeIndex < place.length) {
```
Comment (Atlante45) [inline examples/lobby.js]: places.length

File: examples/lobby.js [hunk 3]
```diff
@@ -252,14 +252,14 @@ function actionStartEvent(event) {
...
+          var actionPlace = places[placeIndex];
           
-          print("Jumping to " + actionLocation.name + " at " + actionLocation.path 
-            + " in " + actionLocation.domain.name + " after click on panel " + panelIndex + " with location index " + locationIndex);
+          print("Jumping to " + actionPlace.name + " at " + actionPlace.address 
+            + " after click on panel " + panelIndex + " with place index " + placeIndex);
           
-          Window.location = actionLocation;
+          Window.location = actionPlace.address;
```
Comment (Atlante45) [inline examples/lobby.js]: Don't know how the new API works but in the print you changed:
actionLocation.path to actionPlace.address
and here:
actionLocation to actionPlace.address

Is that equivalent?

Comment (birarda) [inline examples/lobby.js]: previously we gave the Window.location the whole object - now I'm returning an address and just setting the window location with that

Comment (Atlante45) [inline examples/lobby.js]: Gotcha

**Pull request merged**